openpype/hosts/fusion/plugins/publish/validate_instance_frame_range.py

```diff
@@ -1,17 +1,25 @@
 import pyblish.api
 
-from openpype.pipeline import PublishValidationError
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
 
 
-class ValidateInstanceFrameRange(pyblish.api.InstancePlugin):
+class ValidateInstanceFrameRange(pyblish.api.InstancePlugin,
+                                 OptionalPyblishPluginMixin):
     """Validate instance frame range is within comp's global render range."""
 
     order = pyblish.api.ValidatorOrder
-    label = "Validate Filename Has Extension"
+    label = "Validate Frame Range"
     families = ["render"]
     hosts = ["fusion"]
 
+    optional = True
```

Comment:
Actually no! This validation should not be optional - since by design Fusion should not be capable of rendering ranges outside of the comps global range, which is what this validator validates.

It does **not** validate whether it matches the current asset, but only whether the render range is inside the comp's global frame range.

@EmberLightVFX can likely elaborate more.

Disabling this validation would as far as I know now mean you'd be allowing to render ranges that'd error anyway?

Reply:
Inside the publisher you you have 3 options you can select between.
![image](https://github.com/ynput/OpenPype/assets/49758407/e35fb766-6539-4a99-aa52-220f010b3b05)
`Current asset context` makes sure you render with the correct range for this asset
`From render in/out` lets you render from Fusions in/out points that you manually can set to what ever
`From composition timeline` renders out the current whole comp timeline to what ever you have set it.

This way each saver can have their own render-type so the main render node always render the assets context for example.

If anything a PR where you can select specific in/out points for a render could be welcome. That way if you have to render out a sequence longer than the assets context that you later will re-time in the same comp, saving the in/out points to the saver itself could help to render out the exact same longer range if you need to re-render it for any reason.

Reply:
For context, we actually have an implementation on our repo that allows us to define Custom start/frame per saver as well. I think we implemented that two/three weeks ago.

Reply:
Ohh! PR it! Currently I place sticky notes next to my savers that have custom in/out points 😄

Reply:
@EmberLightVFX here you go: https://github.com/ynput/OpenPype/pull/5968